Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -333,7 +334,20 @@ await _dbContext.TestQuestion
 
         public async Task<TestAC> GetTestByLinkAsync(string link)
         {
+
             var test = await _dbContext.Test.SingleAsync(x => x.Link.Equals(link));
+
+            var testAttendeeDetails = await _dbContext.TestAttendees.Where(x => x.TestId == test.Id).ToListAsync();
+            foreach (var attendee in testAttendeeDetails)
+            {
+                var testLogs = new TestLogs();
+                testLogs = await _dbContext.TestLogs.Where(x => x.TestAttendeeId == attendee.Id).FirstOrDefaultAsync();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable 'testLogs'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1854)